packages/core/src/model-meter.ts

```diff
@@ -52,17 +52,21 @@ export class MeterChartModel extends ChartModel {
 		const options = this.getOptions();
 		const dataValue = this.getDisplayData().value;
 
+		// use max value if the percentage is bigger than 100%
+		const boundedValue = dataValue > 100 ? 100 : dataValue;
```

Comment:
does this result in the incorrect value being shown in the percentage portion of the title?

Reply:
nope! its so that if the num > 100, it will still get the status for the max (100)